deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -854,7 +863,12 @@ def remove_stage(request, name, stage):
             break
 
     if current_env_stage_with_external_id is not None and current_env_stage_with_external_id['externalId'] is not None:
-        environs_helper.delete_nimbus_identifier(request, current_env_stage_with_external_id['externalId'])
+        try:
+            environs_helper.delete_nimbus_identifier(request, current_env_stage_with_external_id['externalId'])
+        except TeletraanException as detail:
+            message = 'Failed to delete Nimbus identifier {}, Error Message: {}'.format(current_env_stage_with_external_id['externalId'], detail)
```

Comment:
Here we can suggest user to verify if the identifier no longer exists. We often see users report that nimbus identifier already exists when they try to create a new stage.

Reply:
I'll update it to: 'Failed to delete Nimbus identifier {}. Please verify that identifier no longer exists, Error Message: {}'